src/lxc/attach.c

```diff
@@ -222,7 +222,7 @@ static int lxc_attach_to_ns(pid_t pid, int which)
 {
 	char path[MAXPATHLEN];
```

Comment:
In the access(path, X_OK) check below, you could just use "/proc/self/ns" and drop this path variable altogether?